### PR TITLE
[WIP] Fixing a couple test for other locale

### PR DESCRIPTION
**Pull request by FoggyFinder**
according to #43

### CODE REVIEW

**Comment by FoggyFinder** [comment]
> AppVeyor build failed

perhaps I miss something in Regex?

**Comment by JohanLarsson** [comment]
Only one failing test now, the build fails if any test fails so not a build error.

File: Gu.Wpf.UiAutomation/AutomationElements/GridCell.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace Gu.Wpf.UiAutomation
     /// </summary>
     public class GridCell : SelectionItemAutomationElement
     {
-        private static readonly Regex NewItemPlaceHolderRegex = new Regex("^Item: {NewItemPlaceholder}, Column Display Index: \\d$", RegexOptions.Singleline | RegexOptions.Compiled);
+        private static readonly Regex NewItemPlaceHolderRegex = new Regex("^.*?{NewItemPlaceholder}, .*?: \\d$", RegexOptions.Singleline | RegexOptions.Compiled);
```
Comment (JohanLarsson) [inline Gu.Wpf.UiAutomation/AutomationElements/GridCell.cs]: Replacing `*.` with `[^:]*` should make the test pass I think.

**Comment by JohanLarsson** [comment]
Awesome, thanks!

**Pull request merged**